src/components/Main/Header/Links/Links.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import LinkElement from "./LinkElement";
+
+import classes from "./Links.module.css";
+
+export default function Links() {
+  return (
+    <nav className={classes.nav}>
+      <Link to="/">
```

Comment:
Better wrap LinkElement by Link and pass to as props, In thi case you will not be wrapp elements every time